test_vm/src/lib.rs

```diff
@@ -660,6 +675,27 @@ impl<'invocation, 'bs> InvocationCtx<'invocation, 'bs> {
         self.resolve_target(&self.msg.to).unwrap().1
     }
 
+    fn invoke_actor(&mut self) -> Result<Option<IpldBlock>, ActorError> {
+        std::panic::catch_unwind(std::panic::AssertUnwindSafe(|| self.invoke())).unwrap_or_else(
```

Comment:
What's this for? I think we view a panic as always a bug in actors, so we want tests to fail.